docs/plugins.md

```diff
@@ -136,18 +143,42 @@ angular.module('yourApp', ['ngVue', 'ngVue.plugins'])
 	})
 ```
 
-## Vuex
+## Root Vue instance props
+Sometimes a plugin requires you to pass extra properties to the root Vue instance in order to access its features on
+any child component. You can use `$ngVueProvider` at a configuration phase of your Angular.js application to
+pass such extra properties. This way you can inject Vuex's `store`, [Vue i81n](https://github.com/kazupon/vue-i18n)'s `i18n`, or your own [custom Vue.js 
```

Comment:
[vue-i18n]

Reply:
I disagree on this one since the official documentation refers to it as `Vue i18n` (no dash):

![screen shot 2018-09-24 at 13 54 14](https://user-images.githubusercontent.com/607262/45950843-a88b4580-c001-11e8-8ac2-071650817af6.png)